ehr_ml/clmbr/__init__.py

```diff
@@ -372,7 +373,80 @@ def test_helper(is_val: bool, sample_size: int) -> float:
                            os.path.join(model_dir, 'best'))
 
 
-def featurize_patients(model_dir: str, extract_dir: str, l: labeler.SavedLabeler) -> Tuple[np.array, np.array, np.array, np.array]:
+def featurize_patients(model_dir: str, extract_dir: str,
+                       patient_ids: Union[List[int], np.array],
+                       day_offsets: Union[List[int], np.array]) -> np.array:
+    """
+    Featurize patients using the given model.
+    This function will use the GPU if it is available.
+    """
+
+    config = read_config(os.path.join(model_dir, "config.json"))
+    info = read_info(os.path.join(model_dir, "info.json"))
+
+    use_cuda = torch.cuda.is_available()
+
+    model = PredictionModel(config, info, use_cuda=use_cuda).to(
+        device_from_config(use_cuda=use_cuda)
+    )
+    model_data = torch.load(os.path.join(model_dir, "best"), map_location="cpu")
+    model.load_state_dict(model_data)
+
+    final_transformation = partial(
+        PredictionModel.finalize_data,
+        config,
+        info,
+        device_from_config(use_cuda=use_cuda),
+    )
+
+    dummy_labels = [0 for _ in patient_ids]
+    data = (dummy_labels, patient_ids, day_offsets)
+
+    loaded_data = StrideDataset(
+        os.path.join(extract_dir, "extract.db"),
+        os.path.join(extract_dir, "ontology.db"),
+        os.path.join(model_dir, "info.json"),
+        data,
+        data
+    )
+
+    patient_id_to_info = defaultdict(dict)
+    for i, (pid, index) in enumerate(zip(patient_ids, day_offsets)):
+        patient_id_to_info[pid][index] = i
+
+    patient_representations = np.zeros((len(patient_ids), config['size']))
```

Comment:
Code like this probably shouldn't use print.

Reply:
Removed in the latest commit. Any reason for this though? It doesn't seem like this function merits its own logging setup.